searchcore/src/vespa/searchcore/proton/server/proton.cpp

```diff
@@ -85,12 +85,24 @@ diskMemUsageSamplerConfig(const ProtonConfig &proton, const HwInfo &hwInfo)
             hwInfo);
 }
 
+size_t
+deriveBackgroundThreads(const ProtonConfig & proton) {
+    size_t threads = std::thread::hardware_concurrency();
+    if (proton.background.threads != 0) {
+        threads = proton.background.threads;
+    }
+    threads = std::max(threads, size_t(proton.summary.log.numthreads));
```

Comment:
Shouldn't we use 'proton.background.threads' here?

Reply:
No, that has been taken into account above.